hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/Utils.java

```diff
@@ -205,7 +205,7 @@ public static String getResourcePath(String resourceName, final String extension
         if (!file.exists()) {
             throw new IllegalArgumentException("Invalid argument: " + path.substring(path.lastIndexOf('/') + 1));
         }
-        return path;
+        return path.equals(file.getPath()) ? path : file.getPath();
```

Comment:
Why not just always return `file.getPath()` for simplicity
```suggestion
        return file.getPath();
```

Reply:
Indeed 🤦🏽‍♂️  Changed now